src/protocol/protocol.yml

```diff
@@ -507,7 +507,7 @@ BrowserContext:
   type: interface
 
   initializer:
-    browserName: string
+    isChromium: boolean
```

Comment:
Why don't we send both? I find browserName pretty useful.

Reply:
`browserName` in the client was only used to distinguish chromium and non-chromium based browsers. Removing it ensures that nobody does `browserName === 'chromum'` when they mean `isChromium`.

If will need it back, it will probably be for some feature that requires larger protocol changes. So let's not have it until we need it.

Reply:
Are you sure about what other language clients do? I am somewhat hesitant breaking them without a good reason.

Reply:
I thought none of them distinguish browsers. I'll check to confirm. I imagine if they are using browserName, they will want to want to fix the same bug and consider electron to be chromium.

Reply:
python reads the browser name, but should switch to `isChromium` after this patch. https://github.com/microsoft/playwright-python/blob/30c5b52cc460f3cce8ba1910682c9d74e0f6f5bf/playwright/_impl/_object_factory.py#L57

Reply:
Same in C#: https://github.com/microsoft/playwright-sharp/blob/afb06a5a3c69f48f524260ad382aab59efd13e77/src/PlaywrightSharp/Page.cs#L793

Java, Go, and Crystal do not appear to distinguish between browsers by name.